interfaces/data/IRssPodcast.ts

```diff
@@ -16,5 +16,5 @@ export interface IRssPodcastValue {
   type: string;
   method: string;
   suggested?: number;
-  valueRecipient: IRssPodcastValueRecipient[];
+  valueRecipients: IRssPodcastValueRecipient[];
```

Comment:
The decorator code sets this as `valueRecipients`:
https://github.com/PRX/Play-Next.js/blob/main/lib/fetch/rss/decoratePodcast.ts#L16